homeassistant/components/ezviz/config_flow.py

```diff
@@ -172,28 +181,23 @@ async def async_step_discovery(self, discovery_info=None):
             }
         )
 
-        if discovery_info is None:
-            return self.async_show_form(
-                step_id="discovery",
-                data_schema=discovered_camera_schema,
-                errors=errors or {},
-            )
-
-        if discovery_info is not None:
-            await self.async_set_unique_id(discovery_info[ATTR_SERIAL])
-            self._abort_if_unique_id_configured()
+        return self.async_show_form(
```

Comment:
It's the confirm step that should show the form.

Reply:
Done.